converter/src/test/resources/negative/iaSectionContainsWrongChild.xml

```diff
@@ -205,6 +205,7 @@
 							<!-- Implied template Measure Reference templateId -->
 							<templateId root="2.16.840.1.113883.10.20.24.3.98"/>
 							<!-- Improvement Activity Performed Reference and Results templateId -->
+							<templateId root="2.16.840.1.113883.10.20.27.3.3"/>
```

Comment:
This'll be a stupid question, but why was this added?

Reply:
This was an accident. Removing.

Reply:
Okay. I added this back. The reason for this is because the IA section validator was failing due to not having any incorrect children error. The reason why is because after removing default decoding, the template ID here was not being decoded anymore and thus not being an incorrect child.